src/locales/cn.yaml

```diff
@@ -1,39 +1,42 @@
 app:
   bedmesh:
     label:
+      box_scale: 显示比例
       flat_surface: 显示参考平面
       mesh_matrix: 网格矩阵
       probed_matrix: 探测点矩阵
-      profile_name: 配置文件名称
-      remove_profile: 删除 %{name} 配置文件
+      profile_name: 配置文件
```

Comment:
I don't speak chinese, but google translate translates this to "configuration file". Are you sure this translation is correct?

Reply:
Looks like "profile" is an alternate translation to this. LGTM!